### ISSUE TITLE
Can't get PlatformTarget from EnvDTE

### ISSUE THREAD

**Issue by bricelam** [dotnet/project-system#669]
Similar to #665, I use to be able to do this...

``` C#
var platformTarget = project.ConfigurationManager.ActiveConfiguration.Properties
    .Item("PlatformTarget")?.Value as string;
```

...but in a .NET Core project, the `PlatformTarget` property doesn't exist.

**Comment by bricelam** [comment]
### Workaround
```C#
var context = (IVsBrowseObjectContext)project;
var unconfiguredProject = context.UnconfiguredProject;
var configuredProject = await unconfiguredProject.GetSuggestedConfiguredProjectAsync();
var properties = configuredProject.Services.ProjectPropertiesProvider.GetCommonProperties();
var platformTarget = await properties.GetEvaluatedPropertyValueAsync("Platform");
```

**Comment by davkean** [comment]
This has been fixed in 15.8 Preview 3.
